docs/issues-and-help.txt

```diff
@@ -0,0 +1,56 @@
+.. _laravel-issues-and-help:
+
+=============
+Issues & Help
+=============
+
+We are lucky to have a vibrant PHP community that includes users of varying
+experience with MongoDB PHP Library and {+odm-short+}. To get support for
```

Comment:
What does `{+odm-short+}` resolve to? I assume this is defined in the related repo with docs configs.

Reply:
Yes it's defined in [this snooty.toml file](https://github.com/10gen/docs-laravel/blob/master/snooty.toml) - it resolves to Laravel MongoDB